lib/grid.rb

```diff
@@ -50,4 +62,34 @@ def coordinates_in_grid?(coordinates)
   def coordinates_unmarked?(coordinates)
     available_coordinates.include?(coordinates)
   end
+
+  def computer_coordinates
+    cells.select { |_, value| value == "O" }.keys.to_set
+  end
+
+  def player_coordinates
+    cells.select { |_, value| value == "X" }.keys.to_set
+  end
+
+  def winner?(cells)
```

Comment:
My first iteration of this feature had a `Rules` class that defined the winning combinations and had the logic for who won. Pretty much every method in that class delegated to `grid`, though, so it seemed like a good idea to move all that stuff into the grid. Now that it's all here, it feels like too much.